docs/ivshmem.md

```diff
@@ -0,0 +1,119 @@
+# Inter-VM shared memory device
+
+The Inter-VM shared memory device (ivshmem) is designed to share a memory region between a guest and the host.
+In order for all guests to be able to pick up the shared memory area, it is modeled by QEMU as a PCI device exposing
+said memory to the guest as a PCI BAR.
```

Comment:
No need to describe how qemu implements.

Reply:
Fixed.